packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -117,10 +118,16 @@ export interface TreeNodeSchemaCore<
 	out Name extends string,
 	out Kind extends NodeKind,
 	out ImplicitlyConstructable extends boolean,
+	out Info = unknown,
 > {
 	readonly identifier: Name;
 	readonly kind: Kind;
-	readonly info: unknown;
+
+	/**
+	 * Data used to define this schema.
+	 * The format depends on the kind of node it is for.
```

Comment:
Can we elaborate on this further? Perhaps an `@example` block?

Reply:
this is arbitrary data in arbitrary format, selected however the user of this type wants, currently mostly desired for writing compile time assertions for type debugging purposes. I don't think there is much of an example that could be useful here without adding some real odd coupling (like making the example of this depend on implementation choices inside of SchemaFactory.object). I agree could use more clarity though, but a formal example block probably won't work for that,.

Reply:
Should be better now